tests/api_tests/test_participant_summary_api.py

```diff
@@ -162,7 +162,8 @@
     "onsiteIdVerificationType": "UNSET",
     "onsiteIdVerificationVisitType": "UNSET",
     "questionnaireOnLifeFunctioning": "UNSET",
-    "aian": False
+    "aian": False,
+    "idVerificationOrigin": "UNSET"
```

Comment:
I don't see an "UNSET" in the `IdVerificationOriginType` enum class definition.  Is the intent to have null/None be the default if no value (REMOTE or ON_SITE) has been assigned yet, or do you need an UNSET = 0 in the enum class that would be the default?

It can impact whether values are returned in the GET API result; a lot of types, if the `participant_summary` column is null, then it's just not included in the returned data.

Reply:
I see what you mean, thanks @pegbertsch ! I will add this as the default type now.

Reply:
Fixed!